spec/services/moves/importer_spec.rb

```diff
@@ -37,34 +37,32 @@
   let!(:profile_one) { create(:profile, person: prisoner_one) }
   let!(:profile_two) { create(:profile, person: prisoner_two) }
 
-  let(:people_importer) { instance_double('People::Importer', call: true) }
+  # let(:people_importer) { instance_double('People::Importer', call: true) }
   let(:alerts_response) do
     [{ offender_no: 'G3239GV', alert_code: 'ACCU9', alert_type: 'MATSTAT' },
      { offender_no: 'G7157AB', alert_code: 'ACCU9', alert_type: 'MATSTAT' },
      { offender_no: 'G7157AB', alert_code: 'ACCU4', alert_type: 'MATSTAT' }]
   end
   let(:offender_numbers_response) { [{ offender_no: 'G3239GV' }, { offender_no: 'G7157AB' }] }
+  let(:prison_numbers_response) do
+    [{ prison_number: 'G3239GV', first_name: 'Bob' },
+     { prison_number: 'G7157AB', first_name: 'Bob' }]
+  end
   let(:personal_care_needs_response) do
     [{ offender_no: 'G3239GV', problem_type: 'MATSTAT', problem_code: 'ACCU9' },
      { offender_no: 'G7157AB', problem_type: 'MATSTAT', problem_code: 'ACCU9' },
      { offender_no: 'G7157AB', problem_type: 'MATSTAT', problem_code: 'ACCU4' }]
   end
 
   before do
-    allow(NomisClient::People).to receive(:get).and_return(offender_numbers_response)
+    allow(NomisClient::People).to receive(:get).and_return(prison_numbers_response)
     allow(NomisClient::Alerts).to receive(:get).and_return(alerts_response)
     allow(NomisClient::PersonalCareNeeds).to receive(:get).and_return(personal_care_needs_response)
-    allow(People::Importer).to receive(:new).and_return(people_importer)
     # create fallback questions for PersonalCareNeeds importer and Alerts importer
     create(:assessment_question, :care_needs_fallback)
     create(:assessment_question, :alerts_fallback)
   end
 
-  it 'calls the People::Importer service twice' do
-    importer.call
-    expect(people_importer).to have_received(:call).twice
-  end
-
```

Comment:
Do we not want to assert the service being called anymore?

Reply:
Nope - see comments above. The test now uses more concrete classes - the fact that the people_importer was called twice (note not even a check for arguments) wasn't a very strong test anyway.